src/services/DfusionService.ts

```diff
@@ -312,12 +317,20 @@ export class DfusionRepoImpl implements DfusionService {
         this._getTcr(),
       ])
 
+      let localOverride: LocalOverride = TOKEN_OVERRIDES[networkId][tokenAddress.toLowerCase()]
+      if (localOverride) {
+        log.info(`Local override found for token address ${tokenAddress} on network ${networkId}:`, localOverride)
```

Comment:
Did TS complain about spreading undefined?

Reply:
Yep it did.